src/main/java/org/elasticsearch/index/query/MoreLikeThisQueryParser.java

```diff
@@ -237,6 +240,21 @@ public Query parse(QueryParseContext parseContext) throws IOException, QueryPars
             }
             // fetching the items with multi-termvectors API
             org.apache.lucene.index.Fields[] likeFields = fetchService.fetch(items);
+            for (int i = 0; i < likeFields.length; i++) {
```

Comment:
This looks good, but shouldn't we build support for `just_name` in the TVs API instead?